h/accounts/subscribers.py

```diff
@@ -1,28 +0,0 @@
-# -*- coding: utf-8 -*-
-from pyramid.events import subscriber
-
-from h.accounts import events
-
-
-@subscriber(events.LoginEvent)
-def login(event):
-    event.request.stats.incr("auth.local.login")
-
-
-@subscriber(events.LogoutEvent)
-def logout(event):
-    event.request.stats.incr("auth.local.logout")
-
-
-@subscriber(events.PasswordResetEvent)
-def password_reset(event):
-    event.request.stats.incr("auth.local.reset_password")
-
-
-@subscriber(events.ActivationEvent)
-def activation(event):
-    event.request.stats.incr("auth.local.activate")
```

Comment:
The `"auth.local.activate"` stat originates from https://github.com/hypothesis/h/pull/1604. Back in 2014 we wanted to know a bunch of key metrics about the service we were running. We're not using these anymore, since the Graphite and Grafana instances they were reported to haven't been running for years